✨ KCP regenerates kubeconfigs before client certs expire (#3140)
kubernetes-sigs/cluster-api: docs/book/src/developer/architecture/controllers/control-plane.md

```

[scale]: https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#scale-subresource

## Kubeconfig management

> Control Plane providers are expected to create and maintain an admin Kubeconfig

Reviewer: ```suggestion
Control Plane providers are expected to create and maintain a Kubeconfig
```
